backoffice/pages/customers/index.tsx

```diff
@@ -27,6 +50,12 @@ const Customers: NextPage = () => {
       .catch((err) => {
         toast.error('Something was wrong! Try later!');
       });
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+    getListCustomer();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Are some of these comments necessary? If not, remove them.

Reply:
Yes. this line is disable warning message from react-hook